vnext/ReactUWP/Modules/DevSupportManagerUwp.cpp

```diff
@@ -73,12 +74,64 @@ void DevSupportManager::LaunchDevTools(const facebook::react::DevSettings &setti
   DownloadFromAsync(facebook::react::DevServerHelper::get_LaunchDevToolsCommandUrl(settings.debugHost)).get();
 }
 
+std::future<void> DevSupportManager::CreatePackagerConnection(const facebook::react::DevSettings &settings) {
```

Comment:
>std::future<void> [](start = 0, length = 17)

Is it possible to avoid using the std::future? It does not offer a continuation. The only way to observe it is to block a thread which is an anti-pattern. 
It is better to use either IAsyncAction or Mso::Future<void>.